external/storm-cassandra/src/main/java/org/apache/storm/cassandra/trident/state/OpaqueTupleStateMapper.java

```diff
@@ -99,6 +98,7 @@ public Values toValues(OpaqueValue<ITuple> tuple) {
             prev = null;
         }
 
+        Long currTx = (Long) values.get(index++);
```

Comment:
I think this introduces a bug. The index is supposed to be 0 here, but the for loops above can increment it. Maybe initialize index to 1 above, and then just do get(0) here.